lxc/copy.go

```diff
@@ -4,33 +4,39 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/codegangsta/cli"
 	"github.com/lxc/lxd"
 	"github.com/lxc/lxd/shared"
-	"github.com/lxc/lxd/shared/gnuflag"
 	"github.com/lxc/lxd/shared/i18n"
 )
 
-type copyCmd struct {
-	ephem bool
-}
+var commandCopy = cli.Command{
+	Name:      "copy",
+	Usage:     i18n.G("Copy containers within or in between lxd instances."),
+	ArgsUsage: i18n.G("[remote:]<source container> [remote:]<destination container>"),
 
-func (c *copyCmd) showByDefault() bool {
-	return true
+	Flags: append(commandGlobalFlags, cli.BoolFlag{
```

Comment:
Is it really safe to use append like this? e.g. fedcff6dff73e4e9416c481529f67aeca09968e2